tests/golden/check-color/check-color.bats

```diff
@@ -11,19 +11,19 @@ load '../helpers'
 
 
 @test "Color flag" {
-  xrefcheck -v --color | base64 | { tr -d '\n'; echo; } > $TEST_TEMP_DIR/out.gold
+  CI=false xrefcheck -v --color | base64 | { tr -d '\n'; echo; } > $TEST_TEMP_DIR/out.gold
```

Comment:
I think it was good, `CI=false` likely does not appear in real life.

Reply:
I think that it is required here because they are checking for the intended behavior when not running in CI, but can be actually running in CI. Otherwise, tests will pass locally but fail during CI.